__tests__/app.test.js

```diff
@@ -66,5 +64,94 @@ describe("/api/reviews", () => {
 				});
 			});
 	});
-
+});
+describe("/api/reviews/:review_id", () => {
+	test("GET 200 - should return an object of the relevant review", () => {
+		const output = {
+			review_id: 1,
+			title: "Agricola",
+			designer: "Uwe Rosenberg",
+			owner: "mallionaire",
+			review_img_url:
+				"https://www.golenbock.com/wp-content/uploads/2015/01/placeholder-user.png",
+			review_body: "Farmyard fun!",
+			category: "euro game",
+			created_at: "2021-01-18T10:00:20.514Z",
+			votes: 1,
+		};
+		return request(app)
+			.get("/api/reviews/1")
+			.expect(200)
+			.then((result) => {
+				expect(result.body).toEqual({ review: output });
+			});
```

Comment:
Flexible matchers should be used for all of the properties except the review_id, to allow for future extention of the endpoint. This test works, but at the slightest change of data it would fail. Try keep tests flexible. Take advantage of expect.objectContaining or .toMatchObject.